dags/synapse-dataset-to-croissant.py

```diff
@@ -308,236 +613,142 @@ def query_snowflake_and_push_croissant_file(root_carrier_context: Dict, dataset_
+            span.set_attribute("airflow.croissant_result", True)
+
+            distribution_files = [{
+                "@type": "FileObject",
+                "@id": "metadata",
+                "contentUrl": f"https://www.synapse.org/Synapse:{dataset_id}.{dataset_version}",
+                "name": "metadata",
+                "description": f"Metadata associated with {dataset.name}",
+                "encodingFormat": "application/csv",
+                "md5": "unknown",
+                "sha256": "unknown"
+            }] + construct_distribution_section_for_files(files_attached_to_dataset, **context)
+
+            record_set = construct_record_set_section_for_files(
+                files_attached_to_dataset=files_attached_to_dataset)
+
+            croissant_file = {
+                "@context": {
+                    "@language": "en",
+                    "@vocab": "https://schema.org/",
+                    "citeAs": "cr:citeAs",
+                    "column": "cr:column",
+                    "conformsTo": "dct:conformsTo",
+                    "cr": "http://mlcommons.org/croissant/",
+                    "rai": "http://mlcommons.org/croissant/RAI/",
+                    "data": {
+                        "@id": "cr:data",
+                        "@type": "@json"
+                    },
+                    "dataType": {
+                        "@id": "cr:dataType",
+                        "@type": "@vocab"
+                    },
+                    "dct": "http://purl.org/dc/terms/",
+                    "examples": {
+                        "@id": "cr:examples",
+                        "@type": "@json"
+                    },
+                    "extract": "cr:extract",
+                    "field": "cr:field",
+                    "fileProperty": "cr:fileProperty",
+                    "fileObject": "cr:fileObject",
+                    "fileSet": "cr:fileSet",
+                    "format": "cr:format",
+                    "includes": "cr:includes",
+                    "isLiveDataset": "cr:isLiveDataset",
+                    "jsonPath": "cr:jsonPath",
+                    "key": "cr:key",
+                    "md5": "cr:md5",
+                    "parentField": "cr:parentField",
+                    "path": "cr:path",
+                    "recordSet": "cr:recordSet",
+                    "references": "cr:references",
+                    "regex": "cr:regex",
+                    "repeated": "cr:repeated",
+                    "replace": "cr:replace",
+                    "sc": "https://schema.org/",
+                    "separator": "cr:separator",
+                    "source": "cr:source",
+                    "subField": "cr:subField",
+                    "transform": "cr:transform"
+                },
+                "@type": "Dataset",
+                "@id": f"{dataset_id}.{dataset_version}",
+                "name": f"{dataset.name}",
+                "description": f"Dataset for {dataset.name}",
```

Comment:
Datasets actually also have a "description" property.  it might be helpful to pull that down?